src/elementary/ReactionNumber/src/register.hpp

```diff
@@ -91,8 +91,333 @@ ReactionNumber::mt_dictionary{
       { 104, ReactionNumber::Entry{ 104, "z,d", { "n,d", "n,deuteron", "z,deuteron" }, { d } } },
       { 105, ReactionNumber::Entry{ 105, "z,t", { "n,t", "n,triton", "z,triton" }, { t } } },
       { 106, ReactionNumber::Entry{ 106, "z,h", { "n,3he", "n,h", "n,he3", "n,helion", "z,3he", "z,he3", "z,helion" }, { h } } },
-      { 107, ReactionNumber::Entry{ 107, "z,a", { "n,a", "n,alpha", "z,alpha" }, { a } } }
-     }; }()
+      { 107, ReactionNumber::Entry{ 107, "z,a", { "n,a", "n,alpha", "z,alpha" }, { a } } },
```

Comment:
I haven't double checked these.

Reply:
There's some indirect testing using the fromEndfReactionNumber function. Not all MTs are there, but I determined the resulting reaction identifier by hand for all tested MTs. It's simple arithmetic so it's doable. There's just a lot of them.

Once everything else is done, we can come back to testing it more rigorously. The faust equivalent of this library has most of that already done.